jni/src/knn_extension/faiss/MultiVectorResultCollector.cpp

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "MultiVectorResultCollector.h"
+#include "knn_extension/faiss/utils/Heap.h"
+#include "knn_extension/faiss/utils/BitSet.h"
+
+namespace os_faiss {
+
+using idx_t = faiss::idx_t;
+
+MultiVectorResultCollector::MultiVectorResultCollector(BitSet* parent_bit_set, const std::vector<int64_t>* id_map)
+: parent_bit_set(parent_bit_set), id_map(id_map) {}
+
+void MultiVectorResultCollector::collect(
+        int k,
+        int& nres,
+        float* bh_val,
+        int64_t* bh_ids,
```

Comment:
can we have some better names for these variable?

Reply:
Following faiss naming convention. Let me refine the comments to explain the parameter.